pcsx2/GS/GSState.cpp

```diff
@@ -652,12 +661,14 @@ void GSState::GIFPackedRegHandlerSTQRGBAXYZF2(const GIFPackedReg* RESTRICT r, u3
 	}
 
 	const GIFPackedReg* RESTRICT r_end = r + size;
+	const GSVector4i ff_mask = GSVector4i::x000000ff();
```

Comment:
These could be constexpr (so it's a single load from memory). Not sure if the constant-returning methods in GSVector4i are, though (they probably should be..)